app/resources/v1/reception_resource.rb

```diff
@@ -20,18 +20,31 @@ def request_attributes=(request_parameters)
 
       @model.request_attributes = request_parameters.map do |request|
         request.permit(request: permitted_request_attributes,
-                       sample: %i[name external_id species], container: %i[type barcode position])
+                       sample: %i[name external_id species study_uuid],
+                       container: %i[type barcode position])
                .to_h
                .with_indifferent_access
       end
     end
 
     def construct_resources!
       @model.construct_resources!
+      publish_messages
     end
 
     def permitted_request_attributes
       [*::Pacbio.request_attributes, *::Ont.request_attributes, *::Saphyr.request_attributes].uniq
     end
+
+    def publish_messages
+      Messages.publish(
+        @model.requests.map(&:sample).flatten,
```

Comment:
Why do we need to flatten?

Reply:
I think we don't because ```sample``` is not a list, so I'll remove it.